### ISSUE TITLE
wrong concrete physics

### ISSUE THREAD

**Issue by kingoscargames** [Wuzzy2/MineClone2-Bugs#231]
concrete powder shouldnt fall through water: https://youtu.be/NUF5KpfSNyw

**Comment by Wuzzy2** [comment]
Yes. This requires me to overwrite the builtin falling node entity, and I currently shy away from touching more entity code. I prefer to not touch this for now, I have already managed to break item entities, we don't need broken falling nodes as well. xD

Lower priority as all MC 1.12 issues are by default.

**Comment by kingoscargames** [comment]
you a trying to clone 1.11 right?

**Comment by Wuzzy2** [comment]
Yes. At least for now. I have set the goal of 1.11 when I started the project simply so I don't have to run after a moving target (which can be exhausting). But that doesn't mean I reject 1.12 features entirely.

**Comment by kingoscargames** [comment]
and Minecraft pc is going to be called Minecraft java edition and some other versions are going to be called just "Minecraft".
